kvbc/src/pruning_handler.cpp

```diff
@@ -192,9 +180,7 @@ bool PruningHandler::handle(const concord::messages::PruneRequest& request,
   }
 
   const auto latest_prunable_block_id = agreedPrunableBlockId(request);
-  // Make sure we have persisted the agreed prunable block ID before proceeding.
-  // Rationale is that we want to be able to pick up in case of a crash.
-  persistLastAgreedPrunableBlockId(latest_prunable_block_id, bftSeqNum);
```

Comment:
Is that replaced by persisting the `PruneRequest` reconfiguration command?

Reply:
yes, but currently we save the command itself under a different key to prevent issues with concord pruing handler